### DISCUSSION TITLE
ブログ記事のタイトルと本文にtypo

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#284] [要望・アイデア・不具合]
https://blog.qiita.com/user-questionnaire-202208/ のタイトルが「Aamazon」になっています。本文中にも同じtypoがあります。

<img width="791" alt="Screen Shot 2022-08-26 at 7 23 43" src="https://user-images.githubusercontent.com/1148320/186779326-2d773987-aefa-4662-988b-f0d9f840606b.png">

P.S.
アンケート、回答しました！

**Comment by kskwtnk** [comment]
@JunichiIto 

失礼いたしました :bow: 
こちら修正が完了しました！
発見いただきありがとうございます :pray:
アンケートの回答もありがとうございます！
